testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -73,8 +73,8 @@ class DockerComposeNamespace(DockerNamespace):
         COMPOSE_FILES_PATH + "/extra/recaptcha-testing/tenantadm-test-recaptcha-conf.yml",
     ]
 
-    NUM_SERVICES_OPENSOURCE = 13
-    NUM_SERVICES_ENTERPRISE = 17
+    NUM_SERVICES_OPENSOURCE = 14
+    NUM_SERVICES_ENTERPRISE = 18
```

Comment:
Reminder: we'll need a corresponding num services update in `mender-docs` [here](https://github.com/mendersoftware/mender-docs/blob/master/07.Administration/02.Production-installation/docs.md) and [here](https://github.com/mendersoftware/mender-docs/blob/master/07.Administration/02.Production-installation/01.Upgrading-from-OS-to-Enterprise/docs.md).

Reply:
Honestly... Maybe there is room to fix this in a better way?

Once this test in mender-docs has integration repo cloned, it is very simple to grep and extract `NUM_SERVICES_OPENSOURCE` and `NUM_SERVICES_ENTERPRISE` from this file to be used in that test. We could even move it to a separate file more obvious like `infra/defs.py` or something like that (it is not docker-compose specific at all)